### PR TITLE
Add examples to pca9685

### PR DESCRIPTION
**Pull request by gsieben**
The entry of the ID at the hub is missing. Therefore it is very confusing for a beginner.

### CODE REVIEW

File: components/output/pca9685.rst [hunk 1]
```diff
@@ -81,6 +82,27 @@ Configuration variables:
   Use this if you have multiple PCA9685s you want to use at the same time.
 - All other options from :ref:`Output <config-output>`.
 
+
+Note
```
Comment (jesserockz) [inline components/output/pca9685.rst]: I don't like the title here being `Note`...
But cannot think of what it should be

**Review by jesserockz — CHANGES_REQUESTED** [review]

File: components/output/pca9685.rst (hunk 1 above)
Comment (gsieben) [reply components/output/pca9685.rst]: Yes. Good question. I tried to stick to the existing structure. Here are a few suggestions:

Usage
Example
Use case

Comment (jesserockz) [reply components/output/pca9685.rst]:
```suggestion
Examples
```

File: components/output/pca9685.rst [hunk 2]
```diff
@@ -81,6 +82,27 @@ Configuration variables:
   Use this if you have multiple PCA9685s you want to use at the same time.
 - All other options from :ref:`Output <config-output>`.
 
+
+Examples
+----
```
Comment (jesserockz) [inline components/output/pca9685.rst]:
```suggestion
--------
```

**Review by jesserockz — DISMISSED** [review]

**Pull request merged**